### ISSUE TITLE
Can we have opset >= 9 for DUC model?

### ISSUE THREAD

**Issue by bbap-7fat** [onnx/models#397]
# Model Request

### Which model would you like to see in the model zoo?
object_detection_segmentation/duc

### Describe why this model is relevant
Since current object_detection_segmentation/duc is opset 7, which keeps spatial as an attribute in operator:BatchNormalization. 
But there is an issue: https://github.com/pytorch/pytorch/pull/9492.
In this PR, it only handle spatial=1, doesn't handle spatial != 1 condition.

In addition, current DUC model has BatchNormalization operator with spatial = 0. Therefore, DUC currently cannot be used.(Or my usage is not correct)

![image](https://user-images.githubusercontent.com/56011928/99040976-0ce43700-25c5-11eb-823b-35fddf31d901.png)
